color/src/colorspace.rs

```diff
@@ -252,6 +295,10 @@ impl Colorspace for Oklab {
     fn scale_chroma(src: [f32; 3], scale: f32) -> [f32; 3] {
         [src[0], src[1] * scale, src[2] * scale]
     }
+
+    fn clip(src: [f32; 3]) -> [f32; 3] {
```

Comment:
Note the style point from #7, this could be `[l, a, b]: [f32; 3]`. I *didn't* apply that completely consistently (see scale_chroma above) but might be a good idea going forward.